src/components/MovieDetails.js

```diff
@@ -1,49 +1,73 @@
 import '../css/MovieDetails.css'
+import React, { Component, Fragment } from 'react'
+import { render } from '@testing-library/react'
+import endpoints from '../endpoints'
 
-const MovieDetails = ({ posters, id }) => {
-  const movie = posters.find(movie => {
-    return movie.id === id
-  })
+class MovieDetails extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      movie: {}
+    }
+  }
```

Comment:
I'll be honest, I was a little sad about changing this to a class based component because I think the constructor can make things look a little messy.

One thing to ponder is the idea of doing our second fetch (with the movie ID) inside the App component, and then passing the whole movie object.  Then MovieDetails can remain a functional component.  We would also be able to destructor outside of the render method again!

I love how many different ways there are to approach these problems 😊